Exercícios/Para casa/entregas/Celina-Oliveira/bank.js

```diff
@@ -0,0 +1,27 @@
+class Bank {
+	bankCode;
+	bankName;
+    #transferTax;
+
+	static createdBanks = [];
+
+	constructor(bankCode, bankName, transferTax) {
+		this.bankCode = bankCode;
+		this.bankName = bankName;
+		this.constructor.createdBanks.push({
+			bankCode: this.bankCode,
+			qtdClients: 0,
+		});
+	}
+
+    get transferTax() {
+        return this.#transferTax; 
+    }
+
+    set transferTax(newTransferTax)  {
```

Comment:
Oi Celina tudo bem? Fiz de uma forma diferente aqui, talvez possa ser mais uma opção, usei o amount como parâmetro e depois atribuí o this.#transferTax a ele.